openpype/pipeline/colorspace.py

```diff
@@ -437,17 +479,72 @@ def get_imageio_file_rules(project_name, host_name, project_settings=None):
 
     # get file rules from global and host_name
     frules_global = imageio_global["file_rules"]
+    activate_global_rules = frules_global.get(
+        "activate_global_file_rules", False)
+
+    if not activate_global_rules:
+        log.info(
+            "Colorspace global file rules are disabled."
+        )
+        return {}
+
     # host is optional, some might not have any settings
     frules_host = imageio_host.get("file_rules", {})
 
     # compile file rules dictionary
-    file_rules = {}
-    if frules_global["enabled"]:
-        file_rules.update(frules_global["rules"])
-    if frules_host and frules_host["enabled"]:
-        file_rules.update(frules_host["rules"])
+    override_global_rules = frules_host.get("override_global_rules")
+    if override_global_rules:
+        return frules_host["rules"]
+    else:
+        return frules_global["rules"]
+
+
+def get_remapped_colorspace_to_native(
+        ocio_colorspace_name, host_name, imageio_host_settings):
+    """Return native colorspace name.
+
+    Args:
+        ocio_colorspace_name (str | None): ocio colorspace name
+
+    Returns:
+        str: native colorspace name defined in remapping or None
+    """
+
+    if not cashed_data.remapping.get(host_name):
+        remapping_rules = imageio_host_settings["remapping"]["rules"]
+        cashed_data.remapping[host_name] = {
+            "to_native": {
+                rule["ocio_name"]: input["host_native_name"]
+                for rule in remapping_rules
+            }
+        }
+
+    return cashed_data.remapping[host_name]["to_native"].get(
+        ocio_colorspace_name)
+
+
+def get_remapped_colorspace_from_native(
+        host_native_colorspace_name, host_name, imageio_host_settings):
+    """Return ocio colorspace name remapped from host native used name.
+
+    Args:
+        host_native_colorspace_name (str): host native colorspace name
+
+    Returns:
+        str: ocio colorspace name defined in remapping or None
+    """
+
+    if not cashed_data.remapping.get(host_name):
+        remapping_rules = imageio_host_settings["remapping"]["rules"]
+        cashed_data.remapping[host_name] = {
+            "from_native": {
+                input["host_native_name"]: rule["ocio_name"]
+                for rule in remapping_rules
+            }
+        }
 
-    return file_rules
+    return cashed_data.remapping[host_name]["from_native"].get(
+        host_native_colorspace_name)
```

Comment:
Even without caching thes efunctions don't seem inheritly slow. Actually, the call would just be:

```python
return imageio_host_settings["remapping"]["rules"].get(host_name, {}).get("from_native", {}).get(host_native_colorspace_name)
```
It's basically only a dictionary lookup. Why cache it?

Reply:
This would not work. We are creating the dictionary from list and there is not `from_native` or `to_native` subcategories in settings.